<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hardware</class>
 <widget class="QFrame" name="Hardware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hardware options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="7" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Expansion Header</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="dischargeDelay">
     <property name="maximum">
      <number>65533</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ADC resolution, bit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>18 ticks = 1 μs</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>Discharge delay, ticks</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="Param2Label">
     <property name="text">
      <string>Cooldown time, ms</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="Param1">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="Param2">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="Param1Label">
     <property name="text">
      <string>Drive time, ms</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="chargeDelay">
     <property name="maximum">
      <number>253</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="adcBits">
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="debouncingTicks">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="revertButton">
       <property name="text">
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Key debouncing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_5">
     <property name="toolTip">
      <string>18 ticks = 1 μs</string>
     </property>
     <property name="text">
      <string>Charge delay, ticks</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QComboBox" name="modeBox"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Pedal debouncing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="pedalDebouncingTicks">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Hardware</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
